mino/httpclient/mod.go

```diff
@@ -0,0 +1,140 @@
+package httpclient
```

Comment:
It looks to me it is doing the job of a proxy, so what do you think about `proxy` ? Then you have the HTTP proxy implementation.

Reply:
also missing tests